content/guides/anomalies.md

```diff
@@ -45,10 +45,10 @@ Start by navigating to the [New Monitor](https://app.datadoghq.com/monitors#/cre
 You should now see something like what's shown above, with a handful of selections that will help determine how sensitive you monitor is to different types of anomalies.
 
 <ol type="a">
-  <li>This number is equivalent to the <code>bounds</code> parameter used in the <code>anomalies</code> function in dashboards; it controls the width of the gray band.</li>
+  <li>This number is equivalent to the <code>bounds</code> parameter used in the <code>anomalies</code> function in dashboards; it controls the width of the gray band. We recommend using a value of 2 or 3.</li>
   <li>If you only care about unusually high or unusually low values, you can choose to only alert on values above or below the bounds.</li>
-  <li>As with other alerts, smaller time windows lead to faster alerting but can yield more false positives.</li>
-  <li>You can change the anomaly detection algorithm used here.</li>
+  <li>We recommend using a window size of at least 15 minutes. (A 30 minute window works well in most cases.) With small windows (<= 10 minutes), metrics often appear noisy, making it difficult to visualize the difference between anomalies and noise. Note that setting this window size to X minutes doesn't require an anomaly to last X minutes before an alert is triggered. You can tune the threshold to control how long an anomaly must last to trigger an alert. For example, with this window size set to 30 minutes, you can can alerted when an anomaly lasts for just five minutes by setting the threshold to 5/30 = 17%.</li>
```

Comment:
This is too much information here. I'd say after the 30 min suggestion, say go to FAQ for more.